README.md

```diff
@@ -171,21 +185,23 @@ api-gateway-kongproxy   10.107.109.1   <nodes>       80:31788/TCP,443:32611/TCP
 We are looking at the port associated with https/443 => 32611
 
 ```
-$ curl -k "https://api.dispatch.local:32611/hello" -H "Content-Type: application/json" -d '{"name": "Jon", "place": "winterfell"}'
+$ curl -k "https://$DISPATCH_HOST:32611/hello" -H "Content-Type: application/json" -d '{"name": "Jon", "place": "winterfell"}'
```

Comment:
the host for api-gateway could be different from dispatch host (especially in a hosted k8s cluster), should we consider to differentiate them?

Reply:
The quickstart section so far only deals with minikube so wanted to keep it simple. We should elaborate the difference in standard installation.